Packs/Base/Scripts/StixParser/StixParser.py

```diff
@@ -242,20 +243,11 @@ def indicators_parser(stix_indicator):
         elif stix_indicator.get("description") == "cve cvss score":
             new_indicator = stix_indicator.get("name")
             if new_indicator:
-                patterns_lists["CVE CVSS Score"].append(new_indicator)
+                patterns_lists.setdefault("CVE CVSS Score", []).append(new_indicator)
```

Comment:
Very nice

Reply:
There's also a `defaultdict` object that you can use ;)